composer.json

```diff
@@ -118,7 +118,6 @@
         "dmore/behat-chrome-extension": "^1.3",
         "dmore/chrome-mink-driver": "^2.7",
         "doctrine/data-fixtures": "^1.4",
-        "ergebnis/composer-normalize": "^2.5",
```

Comment:
Removing this library at it crashes Composer 2:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - ergebnis/composer-normalize[2.5.0, ..., 2.6.0] require ergebnis/json-printer ^3.0.2 -> found ergebnis/json-printer[dev-dependabot/github_actions/ibiqlik/action-yamllint-v3, dev-dependabot/github_actions/actions/cache-v2.1.3, dev-main] but it does not match the constraint.
    - ergebnis/composer-normalize[2.6.1, ..., 2.8.0] require ergebnis/json-printer ^3.1.0 -> found ergebnis/json-printer[dev-dependabot/github_actions/ibiqlik/action-yamllint-v3, dev-dependabot/github_actions/actions/cache-v2.1.3, dev-main] but it does not match the constraint.
    - ergebnis/composer-normalize[2.8.1, ..., 2.11.0] require ergebnis/json-printer ^3.1.1 -> found ergebnis/json-printer[dev-dependabot/github_actions/ibiqlik/action-yamllint-v3, dev-dependabot/github_actions/actions/cache-v2.1.3, dev-main] but it does not match the constraint.
    - Root composer.json requires ergebnis/composer-normalize ^2.5 -> satisfiable by ergebnis/composer-normalize[2.5.0, ..., 2.11.0].
```

We don't use it in CI, only locally.